dockerfiles/ubuntu/is/Dockerfile

```diff
@@ -77,6 +78,6 @@ ENV JAVA_HOME=${JAVA_HOME} \
     WORKING_DIRECTORY=${USER_HOME}
 
 # expose ports
-EXPOSE 4000 9763 9443 
+EXPOSE 4000 9763 9443
 
-ENTRYPOINT ${WORKING_DIRECTORY}/init.sh
+ENTRYPOINT ${WSO2_SERVER_HOME}/bin/wso2server.sh
```

Comment:
@chamindi-a this should not change. The `ENTRYPOINT` should refer to `${WORKING_DIRECTORY}/init.sh`.